diff -upr a/usr/lib/linuxmint/mintMenu/plugins/applications.py b/usr/lib/linuxmint/mintMenu/plugins/applications.py
--- a/usr/lib/linuxmint/mintMenu/plugins/applications.py	2015-04-05 18:30:37.352536150 -0600
+++ b/usr/lib/linuxmint/mintMenu/plugins/applications.py	2015-04-05 18:49:56.490596582 -0600
@@ -1079,35 +1079,35 @@ class pluginclass( object ):
         menuItem = Gtk.SeparatorMenuItem()
         menu.append(menuItem)
         
-        menuItem = Gtk.ImageMenuItem(_("Find Software"))
+        menuItem = Gtk.ImageMenuItem(_("Find Software for Salix"))
         img = Gtk.Image()
         img.set_from_file('/usr/share/linuxmint/mintMenu/search_engines/software.png')
         menuItem.set_image(img)
         menuItem.connect("activate", self.search_mint_software)
         menu.append(menuItem)
         
-        menuItem = Gtk.ImageMenuItem(_("Find Tutorials"))
+        menuItem = Gtk.ImageMenuItem(_("Find Help for Salix"))
         img = Gtk.Image()
         img.set_from_file('/usr/share/linuxmint/mintMenu/search_engines/tutorials.png')
         menuItem.set_image(img)
         menuItem.connect("activate", self.search_mint_tutorials)
         menu.append(menuItem)
         
-        menuItem = Gtk.ImageMenuItem(_("Find Hardware"))
+        menuItem = Gtk.ImageMenuItem(_("Spin Up Another Salix Install"))
         img = Gtk.Image()
         img.set_from_file('/usr/share/linuxmint/mintMenu/search_engines/hardware.png')
         menuItem.set_image(img)
         menuItem.connect("activate", self.search_mint_hardware)
         menu.append(menuItem)
         
-        menuItem =Gtk.ImageMenuItem(_("Find Ideas"))
+        menuItem =Gtk.ImageMenuItem(_("Suggest Ideas for Salix"))
         img = Gtk.Image()
         img.set_from_file('/usr/share/linuxmint/mintMenu/search_engines/ideas.png')
         menuItem.set_image(img)
         menuItem.connect("activate", self.search_mint_ideas)
         menu.append(menuItem)
         
-        menuItem = Gtk.ImageMenuItem(_("Find Users"))
+        menuItem = Gtk.ImageMenuItem(_("Custom-Made Salix Packages"))
         img = Gtk.Image()
         img.set_from_file('/usr/share/linuxmint/mintMenu/search_engines/users.png')
         menuItem.set_image(img)
@@ -1152,31 +1152,31 @@ class pluginclass( object ):
     def search_mint_tutorials(self, widget):
         text = self.searchEntry.get_text()
         text = text.replace(" ", "%20")
-        os.system("xdg-open \"http://community.linuxmint.com/index.php/tutorial/search/0/" + text + "\" &")     
+        os.system("xdg-open \"http://forum.salixos.org/viewforum.php?f=16/" "\" &")     
         self.mintMenuWin.hide()
     
     def search_mint_ideas(self, widget):
         text = self.searchEntry.get_text()
         text = text.replace(" ", "%20")
-        os.system("xdg-open \"http://community.linuxmint.com/index.php/idea/search/0/" + text + "\" &")     
+        os.system("xdg-open \"http://forum.salixos.org/viewforum.php?f=32/" "\" &")     
         self.mintMenuWin.hide()
     
     def search_mint_users(self, widget):
         text = self.searchEntry.get_text()
         text = text.replace(" ", "%20")
-        os.system("xdg-open \"http://community.linuxmint.com/index.php/user/search/0/" + text + "\" &")     
+        os.system("xdg-open \"http://people.salixos.org/" "\" &")     
         self.mintMenuWin.hide()
     
     def search_mint_hardware(self, widget):
         text = self.searchEntry.get_text()
         text = text.replace(" ", "%20")
-        os.system("xdg-open \"http://community.linuxmint.com/index.php/hardware/search/0/" + text + "\" &")     
+        os.system("xdg-open \"http://salixos.org/download.html" "\" &")     
         self.mintMenuWin.hide()
         
     def search_mint_software(self, widget):
         text = self.searchEntry.get_text()
         text = text.replace(" ", "%20")
-        os.system("xdg-open \"http://community.linuxmint.com/index.php/software/search/0/" + text + "\" &")     
+        os.system("xdg-open \"http://slackbuilds.org/" "\" &")     
         self.mintMenuWin.hide()
         
 
