diff -upr a/usr/bin/mintmenu b/usr/bin/mintmenu
--- a/usr/bin/mintmenu	2014-09-30 05:19:50.000000000 -0600
+++ b/usr/bin/mintmenu	2015-04-05 18:16:42.926962748 -0600
@@ -12,4 +12,4 @@ if len(sys.argv) > 1:
         os.system("rm -rf ~/.linuxmint/mintMenu")
         print "All mintMenu settings are now restored to default"
 else:
-    os.system("/usr/lib/linuxmint/mintMenu/mintMenu.py")
+    os.system("/usr/share/linuxmint/mintMenu/mintMenu.py")
